docs/content/what-is-a-schedule-to-start-timeout.md

```diff
@@ -10,24 +10,33 @@ tags:
 import CenteredImage from "../components/CenteredImage.js"
 import RelatedReadList from '../components/RelatedReadList.js'
 
-A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts executing that Activity Task.
+A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts (picks it up from the Task Queue) that Activity Task.
+In other words, it's a limit for how long an Activity Task can be enqueued.
+The moment that the Task is picked by the Worker from the task queue is considered to be the start of the Activity Task for the Schedule To Start timeout and associated metrics to avoid clock difference between the Server and the Worker.
 
 <CenteredImage
 imagePath="/diagrams/schedule-to-start-timeout.svg"
 imageSize="100"
 title="Schedule-To-Start Timeout period"
 />
 
-A [Retry Policy](/docs/content/what-is-a-retry-policy) attached to an Activity Execution retries an Activity Task Execution.
-Thus the Schedule-To-Start Timeout is applied to each Activity Task Execution within an Activity Execution.
+A [Retry Policy](/docs/content/what-is-a-retry-policy) attached to an Activity Execution retries an Activity Task.
+Thus, the Schedule-To-Start Timeout and latency metrics are applied to each Activity Task separately within an Activity Execution.
+
+:::warning "Schedule" in Schedule-To-Start and Schedule-To-Close may reference different moments
+
+"Schedule" in Schedule-To-Start references the scheduling of a single Activity Task (each Activity retry has a separate scheduling moment).
+"Schedule" in Schedule-To-Close references the scheduling of the whole Activity Execution.
+
+:::
```

Comment:
```suggestion
"Schedule" in Schedule-To-Start and Schedule-To-Close both refer to the scheduling of an Activity Task.
Schedule-To-Close refers to the scheduling of the **first** Activity Task in the set of Activity Tasks that make up the Activity Execution.
"Schedule" in Schedule-To-Start refers to the scheduling of **any** Activity Task in the set of Activity Tasks that make up the Activity Execution.
In other words, Schedule-To-Close Timeout is enforced once per Activity Execution, whereas Schedule-To-Start Timeout is enforced for each Activity Task.
This is because a [Retry Policy](/docs/content/what-is-a-retry-policy) attached to an Activity Execution retries an Activity Task.
Thus, the Schedule-To-Start Timeout and latency metrics are applied to each Activity Task separately within an Activity Execution.
```
Let's stay away from admonitions if we can.

@Spikhalskiy perhaps "enforced" is the wrong word here? But just trying to synthesize the point a bit and build on existing information that aims to touch on this point.
The whole thing is slightly verbose now, but hits it from all angles.
We can add some of @dtornow diagrams in a future iteration to help hammer things in.

One thing though is, If we call it out here, then we should make the similar point in the Schedule-To-Close page.

Reply:
Agreed. I recommend changing the bold \*\*first\*\* and \*\*any\*\* to italic (\*first\* and \*any\*) to match the style guide's guidance on emphasis (https://developers.google.com/style/text-formatting).

Reply:
"Enforced" looks pretty correct for me here.

Reply:
@djmagee addressed the "italic" change